packages/storybook/src/documentation/developer/community-component-aanleveren.md

```diff
@@ -0,0 +1,285 @@
+
+> **Waarom een wrapper?**
+>
+> Waarom gebruiken we niet direct het component vanuit de community? Is een LUX component nog wel nodig om het NLDS community component te gebruiken? Kan ik het niet gewoon zelf maken?
+
+## Contact Opnemen
+
+Als allereerste is het belangrijk om contact met ons op te nemen. Dan weten wij waar iedereen mee bezig is en voorkomen we samen dat verschillende teams het zelfde werk gaan uitvoeren.
+
+Wij kijken samen naar onze backlog en maken een story aan als deze nog niet bestaat voor het benodigde component. Daarna helpen wij je met opstarten zodat jij je kan bezighouden met wat voor jou belangrijk is; het component zo snel en zo goed mogelijk aanbieden zodat jij het in gebruik kan nemen.
+
+## Community Component Uitzoeken
+
+Op het [componentenoverzicht van NLDS][nlds-components] staan verschillende componenten. Sommigen zullen anders genaamd zijn dan jij zal verwachten. Neem de lijst dus goed door. Zorg er in ieder geval voor dat je alleen componenten kiest met de status `Community`, `Candidate`, of `Hall of Fame`. Deze statussen komen uit het [NLDS Estafettemodel][nlds-estafettemodel].
+
+### Installeren
+
+De componenten van de volgende partijen uit de NLDS community zijn al beschikbaar in LUX componenten:
+
+- Gemeente Utrecht
+
+#### Nieuwe Community Provider Aansluiten
+
+Kies altijd een web component. Bij gelijke geschiktheid kies je een component van een provider die al is aangeloten. In het geval je component kiest dat niet door een van bovenstaande providers wordt aangeboden moet je deze aansluiten. Bekijk het component op Storybook van de provider en zoek uit hoe je deze kan installeren. Daarna voer je stappen uit:
+
+1. In `/packages/web-components-stencil` voer je het `pnpm add -D xxx` commando uit, waar `xxx` de naam van de package is. Bijvoorbeeld voor Gemeente Utrecht is het `pnpm add -D @utrecht/web-components-library-stencil`;
+1. Voeg de provider toe aan de bovenstaande lijst;
+1. In `packages/web-components-stencil/src/global/app.ts` voeg je de volgende regels toe:
+
+```js
+import { defineCustomElements as defineXXXCustomElements } from "xxx"; // <--
+
+export default function () {
+  //...
+  defineXXXCustomElements(); // <--
+}
+```
+
+## LUX Wrapper Bouwen
+
+Hier begint het echte werk. Met behulp van [StencilJS][stencil] ga je een web component maken die het NLDS community component omsluit en de LUX design tokens erop toepast.
+
+### Scaffolding
+
+Ga naar `packages/web-components-stencil` en voer het commando `pnpm run generate` uit. Gebruik de volgende instellingen:
+
+- Tag name (dash-case): `lux-[component-naam]`
+- Sass format: `*.scss Format`
+- Additional files:
+  - `Stylesheet (.scss)`
+  - `Spec test (.spec.tsx)`
+
+Er worden een aantal bestanden voor je gegenereerd. Je mag overal de `lux` prefix verwijderen, behalve in StencilJS's `@Component` class decorator.
```

Comment:
De bestanden worden nu niet in de goede plek gegenereerd en naam heeft ook nog `lux-` ervoor:

```
$ stencil generate lux-alert

The following files have been generated:
  - ..\..\components\components\lux-alert\lux-alert.tsx
  - ..\..\components\components\lux-alert\lux-alert.scss
  - ..\..\components\components\lux-alert\test\lux-alert.spec.tsx
```

Kunnen we dit tweaken in de stencil config?

Reply:
Heb ik naar zitten kijken maar is niet makkelijk te doen. Vandaar dat we van de week een story gerefined hebben die dit probleem oplost. Die moet alle bestanden op de juiste plek zetten met de juiste scaffold content

Reply:
Ik kan de `srcDir` in `stencil.config.ts` aanpassen zodat die naar de `src` folder gaat maar ik weet zo gauw even niet wat het dan kapot maakt

Reply:
Okay, ik kijk er ook wel ff naar

Reply:
Ik heb de tekst in ieder geval aangepast. Het veranderen van het `stencil generate`-script is out of scope voor deze PR